MetaMask/metamask-extension
# Adding a warning when sending a token to its own contract address

Fixes MetaMask/metamask-extension#9437

<img width="460" alt="Screen Shot 2021-03-02 at 10 13 00 AM" src="https://user-images.githubusercontent.com/8732757/109689887-001f2b00-7b43-11eb-970e-175f6edae3bb.png">

Manual testing steps:  
  - Obtain some test ERC-20 tokens, such as LINK
  - Go to send, enter `0x514910771af9ca656af840dff83e8264ecf986ca` (LINK contract address)
  - Confirm when ETH or other asset is selected to send that the error does not show
  - Confirm the error shows when LINK is the selected asset
